src/pretalx/submission/models/question.py

```diff
@@ -75,6 +75,12 @@ class Question(LogMixin, models.Model):
         verbose_name=_('question type'),
         help_text=_('Do you require an answer from every speaker or for every talk?'),
     )
+    submission_type = models.ManyToManyField(
+        to='submission.SubmissionType',
+        related_name='questions',
+        help_text=_('Is this question limited to some submission types only?'),
```

Comment:
You're missing a `verbose_name` here. Also, it would be worth mentioning that leaving this field empty means including all submission types.